packages/sdks/src/components/content/content.lite.tsx

```diff
@@ -110,7 +111,7 @@ export default function ContentComponent(props: ContentProps) {
           // eslint-disable-next-line @typescript-eslint/ban-ts-comment
           // @ts-ignore
           reactNative: props.blocksWrapper || ScrollView,
-          default: props.blocksWrapper || 'div',
+          default: props.blocksWrapper || DynamicDiv,
```

Comment:
Can you explain this workaround? I'm confused by it.

Also, if absolutely necessary, feel free to keep it, but we'll at least want to scope it to just Angular and keeping the default `'div'` for other frameworks

Reply:
As discussed async, this limitation arises because Angular does not directly support HTML tags (strings) as components within ngComponentOutlet (dynamic components), thereby preventing the use of 'div's. DynamicDiv serves as a wrapper div component that renders its children and assigns all provided attributes to it.